### PR TITLE
added DOM and DGM tileindices for TH

### PR DESCRIPTION
**Pull request by VeronicaKoess**

### CODE REVIEW

**Review by linakrisztian — APPROVED** [review]
Would be good to also add readme, license and Co. But should be fine for now

File: DSM/TH/.~lock.TH_dom_list_bearb.csv# [hunk 1]
```diff
@@ -0,0 +1 @@
+,vkoess,vk,13.07.2023 17:33,file:///home/vkoess/.config/libreoffice/4;
```
Comment (neteler) [inline DSM/TH/.~lock.TH_dom_list_bearb.csv#:1]: Please remove this lock file from the PR.

File: DSM/TH/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- `license.txt`
+- data can be used citing the „Datenlizenz Deutschland - Namensnennung - Version 2.0“
+- see file for complete license
+
+Links:
+- [metadata, license, ...](https://www.geoportal-th.de/de-de/Downloadbereiche/Download-Offene-Geodaten-Th%C3%BCringen/Download-H%C3%B6hendaten)
+- [download data](https://www.geoportal-th.de/de-de/Downloadbereiche/Download-Offene-Geodaten-Th%C3%BCringen/Download-H%C3%B6hendaten)
+~                                                                    
+
+
```
Comment (neteler) [inline DSM/TH/README.md:30]:
```suggestion

```

File: DSM/TH/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+TH:
+
+# Thüringen (TH) Tile index of Digital Surface Model from Brandenburg Geobasis data
+
+* Geoportal data: image based Digital Surface Model, 1m resolution,  EPSG:25832
+* [Scripts](https://github.com/mundialis/tile-indices/tree/main/DSM/TH) to create the tile index
+* Download link: https://github.com/mundialis/tile-indices/blob/main/DSM/TH/TH_DOM_tileindex.gpkg.gz
+~
```
Comment (neteler) [inline DSM/TH/README.md:38]:
```suggestion
```

File: DSM/TH/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+
+Generate Tile_index with csv file:
+
+
+- the .csv file with downloadlinks of every tile had to be created manually. The download links were retrieved via an atom feed, published here: https://www.geoportal-th.de/de-de/Downloadbereiche/Download-Offene-Geodaten-Th%C3%BCringen/Download-H%C3%B6hendaten
+
+- as the listed zip files cannot be recognized as a supported file format by gdal ("error 4"), the respecitve xyz files in the zip folders must be specified.
+Accordingly, the csv list was manually changed to e.g: /vsizip/vsicurl/https://geoportal.geoportal-th.de/hoehendaten/DOM/dom_2014-2019/dom1_561_5610_1_th_2014-2019.zip/dom1_561_5610_1_th_2014-2019.xyz.
+Furthermore, defective tiles, that were lying outside the federal state were excluded.
```
Comment (neteler) [inline DSM/TH/README.md:16]:
```suggestion

Generate Tile_index with csv file:

- the .csv file with download links of every tile had to be created manually. The download links were retrieved via an atom feed, published here: https://www.geoportal-th.de/de-de/Downloadbereiche/Download-Offene-Geodaten-Th%C3%BCringen/Download-H%C3%B6hendaten

- as the listed zip files cannot be recognized as a supported file format by GDAL ("error 4"), the respective xyz files in the zip folders must be specified.
Accordingly, the csv list was manually changed to e.g: /vsizip/vsicurl/https://geoportal.geoportal-th.de/hoehendaten/DOM/dom_2014-2019/dom1_561_5610_1_th_2014-2019.zip/dom1_561_5610_1_th_2014-2019.xyz.

Furthermore, defective tiles, that were lying outside the federal state were excluded.
```

File: DSM/TH/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+- based on the output csv, a tileindex.gpkg can be generated by running: gdaltindex -f GPKG TH_DOM_tileindex.gpkg --optfile TH_dom_list_bearb.csv
```
Comment (neteler) [inline DSM/TH/README.md:18]:
```suggestion
- based on the output csv, a tileindex.gpkg can be generated by running: `gdaltindex -f GPKG TH_DOM_tileindex.gpkg --optfile TH_dom_list_bearb.csv`
```

File: DTM/TH/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+Generate Tile_index with csv file:
+
+
+- the .csv file with downloadlinks of every tile had to be created manually. The download links were retrieved via an atom feed, published here: https://www.geoportal-th.de/de-de/Downloadbereiche/Download-Offene-Geodaten-Th%C3%BCringen/Download-H%C3%B6hendaten
+
+- as the listed zip files cannot be recognized as a supported file format by gdal ("error 4"), the respecitve xyz files in the zip folders must be specified.
+Accordingly, the csv list was manually changed to e.g: /vsizip/vsicurl/https://geoportal.geoportal-th.de/hoehendaten/DGM/dgm_2014-2019/dgm1_562_5609_1_th_2014-2019.zip/dgm1_562_5609_1_th_2014-2019.xyz
+
+- based on the output csv, a tileindex.gpkg can be generated by running: gdaltindex -f GPKG TH_DOM_tileindex.gpkg --optfile TH_dgm_list.csv
```
Comment (neteler) [inline DTM/TH/README.md:17]:
```suggestion


Generate Tile_index with csv file:

- the .csv file with download links of every tile had to be created manually. The download links were retrieved via an atom feed, published here: https://www.geoportal-th.de/de-de/Downloadbereiche/Download-Offene-Geodaten-Th%C3%BCringen/Download-H%C3%B6hendaten

- as the listed zip files cannot be recognized as a supported file format by GDAL ("error 4"), the respective xyz files in the zip folders must be specified.
Accordingly, the csv list was manually changed to e.g: /vsizip/vsicurl/https://geoportal.geoportal-th.de/hoehendaten/DGM/dgm_2014-2019/dgm1_562_5609_1_th_2014-2019.zip/dgm1_562_5609_1_th_2014-2019.xyz

- based on the output csv, a tileindex.gpkg can be generated by running: `gdaltindex -f GPKG TH_DOM_tileindex.gpkg --optfile TH_dgm_list.csv`
```

File: DTM/TH/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+
+TH:
+
+# Thüringen (TH) Tile index of Digital Terrain Model from Brandenburg Geobasis data
+
+* Geoportal data: image based Digital Terrain Model, 1m resolution,  EPSG:25832
+* [Scripts](https://github.com/mundialis/tile-indices/tree/main/DSM/TH) to create the tile index
+* Download link: https://github.com/mundialis/tile-indices/blob/main/DSM/TH/TH_DOM_tileindex.gpkg.gz
+~
```
Comment (neteler) [inline DTM/TH/README.md:37]:
```suggestion

TH:

# Thüringen (TH) Tile index of Digital Terrain Model from Brandenburg Geobasis data

* Geoportal data: image based Digital Terrain Model, 1m resolution,  EPSG:25832
* [Scripts](https://github.com/mundialis/tile-indices/tree/main/DSM/TH) to create the tile index
* Download link: https://github.com/mundialis/tile-indices/blob/main/DSM/TH/TH_DOM_tileindex.gpkg.gz
```

**Pull request merged**